balancer-design.typ

```diff
@@ -17,3 +17,46 @@ In Rust, packages are called "crates". The Balancer and Harness is split into mu
 	image("figures/balancer-crates.svg"),
 	caption: "Package diagram showing the Balancer and Harness crates.",
 ) <Figure::balancer-crates>
+
+== Dependencies
+
+#let dependencies = toml("Cargo.toml")
+
+#figure(
+	table(
+		[*Dependencies*],
+		[#dependencies.workspace]
+	),
+	caption: "External dependencies required by the balancer"
+)
+
+#table(
+	[*External Dependencies*],
+	[*Package Name/Description*],
+	[anyhow: Error handling],
+	[async-trait: Support for asyncronous functions in traits],
```

Comment:
You can work these descriptions into the other table if you really want to, but it's not necessary.